versioned_docs/version-2.7/integrations-in-rancher/cloud-marketplace/cloud-marketplace-payg-integration.md

```diff
@@ -0,0 +1,261 @@
+
+We have two listings for Rancher Prime, a EU/EMEA version and a non-EU, non-UK version, due to different regulations in different locations. Whichever listing you choose, the images and support offered are identical. You should pick the listing for the location of your cloud provider account.
+
+#### Are these listings available in all countries?
+
+No. The PAYG listing isn't available in all countries.
+
+Your billing country is based on the cloud provider Account ID for the deployment.
+
+Please refer to [Geographical availability](https://documentation.suse.com/sle-public-cloud/all/html/public-cloud/countrylist.html#) for a list of countries that can and cannot access Rancher Prime via the AWS and Azure cloud marketplaces.
+
+#### My cloud provider account is in the USA, but I want to deploy Rancher in another region, a region that is in a country where I currently cannot transact Rancher Prime. Is this possible?
+
+Yes. As long as your account is billed to one of the allowed countries, it is possible to deploy Rancher in any region.
+
+#### Is this listing available in China?
+
+While it is not possible to transact or bill Rancher Prime in China, it is possible to deploy into regions in China.
+
+#### Can I still deploy Rancher using the "Rancher Setup" listing from the AWS Marketplace?
+
+The Rancher Setup listing is no longer available via AWS Marketplace. Customers should deploy an EKS Cluster to host Rancher. Follow the steps in this [guide](../../getting-started/installation-and-upgrade/install-upgrade-on-a-kubernetes-cluster/rancher-on-amazon-eks.md) except for the Rancher installation. The Rancher product installation should be carried out as per [installing the Rancher Prime PAYG offering on Amazon's AWS Marketplace](../cloud-marketplace/aws-marketplace-payg-integration/installing-rancher-prime.md).
+
+### Billing
+
+#### I have an existing Rancher Prime subscription. Can I use this on the cloud?
+
+BYOS (Bring Your Own Subscription) Rancher deployments are supported on the cloud; however, you won't be billed for your BYOS via the cloud marketplace. Once the existing subscription term ends, you can purchase Rancher Prime via the cloud marketplace and reconfigure your cluster to support monthly billing via the cloud.
+
+#### I have an existing Rancher Subscription purchased via the Rancher Premium Support listing on AWS. Is this transferable to the new model?
+
+No. A new deployment of Rancher Prime is required to benefit from the new monthly billing model.
+
+#### I have an existing deployment covered by a Rancher subscription; can I use this new marketplace listing for new deployments?
+
+Yes. The listing works independently of your existing subscriptions. Please remember that support processes may be different for deployments using your existing subscription and deployments billed via the cloud marketplace.
+
+#### Tell me more about how the billing for Rancher Prime works for the cloud marketplace.
```

Comment:
```suggestion
#### Can you tell me more about how the billing for Rancher Prime works for the cloud marketplace?
```

Phrasing as a question since it's a FAQ.